tnb/base_client.py

```diff
@@ -35,14 +35,14 @@ def send_request_from_url(self, method, url, **kwargs):
 
         return response.json()
 
-    def fetch(self, resource, **kwargs):
+    def fetch(self, resource: str, **kwargs: object) -> object:
```

Comment:
The parameter `kwargs` is a `dict` instead of an `object`.

This method returns either `dict` or a `list`.

**Note**: 
- The method documentation is also incorrect. 
- For this change you will also need to import `Union` with: 
```python
from typing import Union
```
```suggestion
    def fetch(self, resource: str, **kwargs: dict) -> Union[dict, list]:
```